### ISSUE TITLE
What is the final f1 performance?

### ISSUE THREAD

**Issue by guotong1988** [brightmart/bert_language_understanding#3]
![image](https://user-images.githubusercontent.com/4702353/47703986-07755780-dc5d-11e8-9c63-94c90457984a.png)

https://github.com/brightmart/bert_language_understanding#performance

What is the final f1 performance?

Thank you!

**Comment by xwzhong** [comment]
我也想问问，为什么你只展示了特定epoch下的比较？使用pretrain和不使用pretrain最终的F1又是多少呢？因为pretrain确实是能在前期加快收敛，但最终的结果可能反倒变低。

**Comment by brightmart** [comment]
## Performance 

MIDDLE SIZE DATASET(<a href='https://pan.baidu.com/s/1HUzBXB_-zzqv-abWZ74w2Q'>cail2018</a>, 450k)

Model                        | TextCNN(No-pretrain)| TextCNN(Pretrain-Finetuning)| Gain from pre-train 
---                          | ---                 | ---                         | -----------     
F1 Score after 1 epoch       |  0.09               | 0.58                        |  0.49        
F1 Score after 5 epoch       |  0.40               | 0.74                        |  0.35    
F1 Score after 7 epoch       |  0.44               | 0.75                        |  0.31                         
F1 Score after 35 epoch      |  0.58               | 0.75                        |  0.27                                                
Training Loss at beginning   |  284.0              | 84.3                        |  199.7             
Validation Loss after 1 epoch|  13.3               | 1.9                         |  11.4                 
Validation Loss after 5 epoch|  6.7                | 1.3                         |  5.4                              
----------------------------------------------------------------------------------------------
Notice: 

  a.fine-tuning staged completed training after just running 7 epoch as max epoch reached to 35. 

     in fact, pre-train model start training from epoch 27 where pre-train stage ended.

  c.f1 Score reported here is on validation set, an average of micro and macro of f1 score. 

  d.f1 score after 35 epoch is reported on test set.

@guotong1988 @xwzhong  performance updated. please check.
